models/request/opreturn/output.go

```diff
@@ -0,0 +1,10 @@
+package opreturn
+
+type Output struct {
+	DataType DataType `json:"dataType,omitempty"`
+	Data     []string `json:"data"`
+}
+
+func (o Output) GetType() string {
+	return "op_return"
```

Comment:
In the future, maybe, we could use some constant/enum here.

Reply:
I'm not sure yet if this is needed/useful, those constant/enums doesn't prevent you from using custom value (for string you can assign a string, for iota you can assign any int).